Sources/UIViewExtensions.swift

```diff
@@ -263,6 +263,8 @@ extension UIView {
     }
 }
 
+private let UIViewTransitionDuration: TimeInterval = 0.45
```

Comment:
I've wrote UIViewDefaultTransitionDuration, to be in context like these  UIViewAnimationDuration, UIViewAnimationSpringDamping UIViewAnimationSpringVelocity